Automattic/jetpack
# CRM: Add several new currencies to the CRM currency dropdown

## Proposed changes:

* This is a partial fix to this issue - https://github.com/Automattic/zero-bs-crm/issues/2629 - manually adding in a requested currency (plus some additional currencies)
* The currency symbol and strings are added to the currency dropdown and can be selected for display on invoices / client portal.
* The added currencies are Azerbaijan Manat('₼', 'AZN') and Kazakhstani Tenge ('₸', 'KZT'), plus El Salvador Colon ('$', 'SVC'), Tuvalu Dollar ('$', 'TVD') and Zimbabwe Dollar ( '$', 'ZWD' ).

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2629

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* With this PR checked out, on a test site open up the CRM Settings > Locale page: `/wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=locale`. From the currency dropdown, you will see five new options: "₼ (AZN)" and "₸ (KZT)", plus also "$ (SVC)" , "$ (TVD)" and "$ (ZWD)"
* Select these one by one, and check the currency symbol shows correct when viewing all invoices, and viewing in the portal. Note that pdfs don't render many currencies correctly, including the first two new ones, and that is a separate issue (see https://github.com/Automattic/zero-bs-crm/issues/1782#issuecomment-2057184583)
